src/styles/blocks/price.scss

```diff
@@ -0,0 +1,29 @@
+.price {
+  margin-bottom: 16px;
+  display: flex;
+  justify-content: space-between;
+
+  &__text {
+    color: #616070;
+    display: flex;
+    flex-direction: row;
+    width: 31px;
+    height: 18px;
+    font-family: $main-font-famaly;
```

Comment:
There is a typo in the variable name '$main-font-famaly'. It should be '$main-font-family'. Ensure that the variable is correctly defined and imported.